vnet/lib/vnet.rb

```diff
@@ -11,6 +11,10 @@
 require 'json'
 require 'ostruct'
 
+if defined?(::Celluloid)
+  raise "Celluloid module loaded before Vnet."
```

Comment:
What would the user be expected to do in this case? Just restart and hope the race condition doesn't occur?

Reply:
It's not a race condition, it's to catch if the programmer has accidentally added celluloid before vnet.